stablehlo/tests/interpret_if.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: if
+func.func @if() -> (tensor<2xi64>, tensor<2xi64>) {
+  %pred = stablehlo.constant dense<false> : tensor<i1>
+  %result0, %result1 = "stablehlo.if"(%pred) ({
+    %0 = stablehlo.constant dense<0> : tensor<2xi64>
+    "stablehlo.return"(%0, %0) : (tensor<2xi64>, tensor<2xi64>) -> ()
```

Comment:
How about using pretty syntax for stablehlo.return? This will make things easier to write and read.